script/build.mjs

```diff
@@ -0,0 +1,52 @@
+import assert from 'node:assert/strict'
+import fs from 'node:fs/promises'
+import {fetch} from 'undici'
```

Comment:
For scripts like this I prefer using experimental global fetch. If you strongly prefer `undici` though, that’s fine too.

Reply:
doesn’t work with TS yet as far as I know, hence undici

Reply:
This file isn't TypeScript checked as part of the build.

If you add it to part of the type checks, then https://github.com/DefinitelyTyped/DefinitelyTyped/issues/60924#issuecomment-1353371196 would be of interest, there are a few options which use undici as a type only dependency for the declarations.
Or we could pull in fetch from the dom libraries since for our intents and purposes the two versions are the same.

Reply:
Then we need to check it as part of the build.
I’ll look into it, but the TS build is broken locally.

I strongly prefer the current state, where there are no workarounds in place, and things work without global types or DOM types.